test/Microsoft.AspNet.Mvc.Core.Test/AntiXsrf/AntiForgeryWorkerTest.cs

```diff
@@ -379,6 +379,29 @@ public void Validate_FromValidStrings_TokensValidatedSuccessfully()
             context.TokenProvider.Verify();
         }
 
```

Comment:
Add a test that makes sure interaction between `Preset`, `FormTagHelper` and `AntiForgeyToken` works as expected (cookies are not overriden, POST works fine)